docs/user-guide/liveness/README.md

```diff
@@ -66,11 +66,13 @@ The [http-liveness.yaml](http-liveness.yaml) demonstrates the HTTP check.
       httpGet:
         path: /healthz
         port: 8080
+        httpHeaders:
+          X-Custom-Header: [ Awesome ]
```

Comment:
HTTP lets you specify headers multiple times. Generally I find this feature very annoying, but we definitely should support it.